build/Helix/ProcessHelixFiles.ps1

```diff
@@ -51,13 +51,20 @@ function Log-Error
 function Append-HelixAccessTokenToUrl
 {
     Param ([string]$url, [string]$token)
-    if($url.Contains("?"))
+    if($token)
     {
-        $url = "$($url)&access_token=$($token)"
+        if($url.Contains("?"))
+        {
+            $url = "$($url)&access_token=$($token)"
+        }
+        else
+        {
+            $url = "$($url)?access_token=$($token)"
+        }
     }
     else
     {
-        $url = "$($url)?access_token=$($token)"
+        Write-Host "The Helix Access Token was empty, returning the URL as is."
```

Comment:
This will generate a lot of noise in the output since it will be called for every file that is downloaded.